pallets/dao/Cargo.toml

```diff
@@ -4,104 +4,105 @@ version = '0.7.0'
 description = 'Custom pallet for creation decentrilize autonomous organization.'
 authors = ['UNIVERSALDOT FOUNDATION <https://github.com/UniversalDot>']
 homepage = 'https://universaldot.foundation'
-edition = '2018'
+edition = '2021'
 license = 'Apache-2.0'
 publish = false
 repository = 'https://github.com/UniversalDot/pallets'
 
 [package.metadata.docs.rs]
 targets = ['x86_64-unknown-linux-gnu']
 
+# dependencies
 [dependencies.codec]
 default-features = false
 features = ['derive']
 package = 'parity-scale-codec'
-version = '2.0.0'
-
-[dependencies.sp-std]
-default-features = false
-git = 'https://github.com/paritytech/substrate.git'
-tag = 'monthly-2021-11'
-version = '4.0.0-dev'
-
-[dependencies.sp-core]
-default-features = false
-git = 'https://github.com/paritytech/substrate.git'
-tag = 'monthly-2021-11'
-version = '4.0.0-dev'
+version = '3.0.0'
 
 [dependencies.frame-benchmarking]
 default-features = false
```

Comment:
Should we use consisted inline format everywhere just like you have done in node/Cargo.toml?

Reply:
Switched to inline dependency style for pallets.